src/components/Modal/BaseModal.tsx

```diff
@@ -57,7 +58,8 @@ function BaseModal(
     const theme = useTheme();
     const styles = useThemeStyles();
     const StyleUtils = useStyleUtils();
-    const {isSmallScreenWidth, windowWidth, windowHeight} = useWindowDimensions();
+    const {windowWidth, windowHeight} = useWindowDimensions();
+    const {isSmallScreenWidth} = useResponsiveLayout();
```

Comment:
Is there a reason we are not using `shouldUseNarrowLayout` here and several places?

Reply:
@rayane-djouah yes but it's kind of used most of the places? 😄